test/hipmerUtils_test.py

```diff
@@ -41,6 +41,7 @@ def test_validate(self):
                 "diploid": "low"
             },
             'usedebug': 1,
+            'assembly_size_filter': 1200,
```

Comment:
Shouldn't the output be different if the data is now filtered? If you can delete a parameter from a test input and the test doesn't change, that parameter isn't tested.

Reply:
This should change the output data. All except for 1 contig should be filtered away using a 1200 bp cutoff.

Reply:
But based on the test it doesn't. I'd expect some assertion about the output to change, but there is no assertion change.

Reply:
Ah, I was thinking of the other test. Let me take a look.